doc/guide_to_parser_writing.md

```diff
@@ -0,0 +1,86 @@
+# How to write a DefectDojo parser
+
+> All commands assume that you're located at the root of the django-DefectDojo cloned repo.
+
+## Pre-requisites
+- You have forked https://github.com/DefectDojo/django-DefectDojo and cloned locally.
+- Checkout `dev` and make sure you're up to date with the latest changes.
+- It's advised that you create a dedicated branch for your development, such as `git checkout -b parser-name` yet that's up to you.
+
+It is probably easier to use the docker-compose stack (and benefit from the hot-reload capbility for uWSGI).
+Set up your environment to use the dev or ptvsd environment, such as:
+
+`$ docker/setEnv.sh dev`
+or
+`$ docker/setEnv.sh ptvsd` (allows to set breakpoints in uWSGI)
+
+Please have a look at [DOCKER.md](../DOCKER.md) for more details.
+
+### docker images
+You'd want to build your docker images locally, and eventually pass in your local user's `uid` to be able to write to the image (handy for database migration files). Assuming your user's `uid` is `1000`, then:
+
+`$ docker-compose build --build-arg uid=1000`
+
+## Which files do you need to modify?
+
+| File                                          | Purpose
+|-------                                        |--------
+|`dojo/fixtures/test_type.json`                 | Django fixture for the type of scan.
```

Comment:
Random community member here. 👋 I think we might want to explain this bit, and what exactly to do ("increment from the last existing fixture by incrementing the primary key, aka the `pk` field in the fixture JSON") as in my team that is the part most people forget, and most people run into issues with merge conflicts.

Not my PR or I would add something elsewhere, just not in the purpose. Not sure that helps, excuse the intrusion!

Reply:
if people are creating parsers only for their own fork and not planning to push it upstream, it could be an advice to use a `pk` in another range to avoid conflicts maybe, i.e. use `pk` 1001, 1002, etc.